controls/radcombobox/features/textsearch.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Text Search
+page_title: Text Search
+description: Text Search
+slug: radcombobox-features-textsearch
+tags: textsearch
+published: True
+position: 2
+---
+
+# Text Search Functionality
+
+The __RadComboBox__ control supports rich text search functionality. The __TextSearch__ class and the __TextSearchMode__ property of the RadComboBox can be used in order to regulate how the control behaves when it receives user input.
+
+The purpose of this article is to show the properties of the TextSearch class and the TextSearchMode property. The following sections are included:
+
+* [TextSearchMode](#textsearchmode)
+
+* [Using TextSearch.TextPath](#using-textsearchtextpath)
+
+* [Using TextSearch.AutoCompleteTimeout](#using-textsearchautocompletetimeout)
+
+## TextSearchMode
+
+There are four __TextSearchMode__ types that can work with the RadComboBox:
+
+* __Contains__ - only items containing the typed text will be displayed in the drop-down part, case insensitive.
+
+* __ContainsCaseSensitive__ - only items containing the typed text will be displayed in the drop-down part, case sensitive.
+
+* __StartsWith__ - only items, which text starts with the typed text, will be shown in the drop-down part, case insensitive.
+
+* __StartsWithCaseSensitive__ - only items, which text starts with the typed text, will be shown in the drop-down part, case sensitive.
+
+>The __ContainsCaseSensitive__ and __StartsWithCaseSensitive__ are available since Q1 2010 SP1.
```

Comment:
Shouldn't we remove this information? It creates the impression that the article has not been updated for quite a long time.